go/probe/internal/hades/hades.go

```diff
@@ -30,19 +30,22 @@ type Actions interface {
 }
 
 func CreateProbe(token, hq string) *Probe {
-	wd, err := util.FindWorkingDirectory()
-	if err != nil {
-		return nil
-	}
-	return &Probe{
-		signals:       make(chan bool),
-		token:         token,
-		hq:            strings.TrimSuffix(hq, "/"),
-		dos:           strings.ToLower(fmt.Sprintf("%s-%s", runtime.GOOS, runtime.GOARCH)),
-		sleep:         14400 * time.Second,
-		cwd:           wd,
-		commandTimout: 2 * time.Second,
-	}
+    wd := os.Getenv("PRELUDE_DIR")
+    if wd == "" {
+        var err error
+        if wd, err = util.FindWorkingDirectory(); err != nil {
```

Comment:
This is different from our other probes.. I could see either the probe finding a writable dir or getting it as an env var but not both.

Reply:
I thought the point of this issue is to be able to overwrite the directory with the env var for the special cases that the directory we chose isn't writable/ executable? But us choosing should still be the default?

Reply:
Yep-- but this appears to not be using a temp dir but instead hunting for a writable directory. I'm reviewing on a phone so could be wrong (can't see the internals of that util func).

Reply:
It does look like FindWorkingDirectory() walks from cwd to find a writable directory. No guarantee there's a writable directory to be found from cwd, but if there is, this function should find it

Obviously no other probe does this right now. Do we need all the probes to the same behavior, and is that behavior like this or like the others (default to /tmp)?

Reply:
Per our other thread; it makes most sense for the probe to write tests wherever it lives (or tmp) but if someone wants to specify a directory, the installer should have the logic to put the probe in a writable directory.